src/main/content/_assets/js/guides.js

```diff
@@ -9,11 +9,12 @@
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
 
-$(document).ready(function() {
+$(window).on('load', function() {
```

Comment:
The `ready` event occurs when HTML/DOM is fully loaded, while `load` event occurs when all resources are fully loaded.